scaffold/cmd/templates/source/.gitignore.tpl

```diff
@@ -0,0 +1,22 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
```

Comment:
need to ignore vendor

Reply:
Can you please provide more context on your comment?
This is based on the GitHub template https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Go.gitignore#L18, and to my understanding subjective based on the use case.

Also should this be blocking for this PR?

Reply:
I'll go ahead a merge this and we can do a follow up PR if it makes sense

Reply:
Basically not ignoring vendor means that you commit the dependencies into your repo. It's not the case here.

Reply:
Got it, I think `vendor` is subjective (based on a quick research) as if you want to guarantee future reproducible builds you'll have to commit the vendor directory.
I tend to go with what ever the default is in GitHub mostly based on trust 🙃